docs/sources/developers/plugins/backend/debug-your-backend-plugin.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Debug your backend plugin"
+keywords = ["grafana", "plugins", "backend", "plugin", "backend-plugins", "documentation"]
+type = "docs"
++++
+
+# Debug your backend plugin
```

Comment:
I'm not sure this title is accurate. From the title, I expected steps to follow to debug my plugin. The actual instructions seem to be for adding a debugger to the plugin, specifically Delve.

Reply:
"Configure Delve to debug your backend plugin"? @alexanderzobnin